build-system/babel-plugins/babel-plugin-transform-fix-leading-comments/test/fixtures/transform/basic/output.js

```diff
@@ -15,7 +15,7 @@
  */
 
 /**
- * @protected {!Object<string, !ChildDef>|null}
+ * @protected {?Object<string, !ChildDef>}
```

Comment:
`Object` is implicitly nullable
```suggestion
 /** @protected {Object<string, !ChildDef>} */
```

Reply:
It is. But if you [read the docs](https://github.com/google/closure-compiler/wiki/Types-in-the-Closure-Type-System#the-javascript-type-language) under "Nullable Type", they recommend being explicit and including the `?` anyway.